semantic_release/__init__.py

```diff
@@ -3,10 +3,6 @@
 __version__ = '4.1.1'
 
 
-from .errors import UnknownCommitMessageStyleError  # noqa; noqa
```

Comment:
Why are these removed? This is a breaking change, since this will break imports of errors directly from `semantic_release`. Can you please add them back?

Reply:
Yes, I removed them as these caused flake8 errors (see #129), and tests are passing when removing these imports. So it is not clear what pb this causes.

Can you clarify the pb with an actual test or specific example?

Reply:
hi @relekang ,

A kind reminder. Could you clarify please? Would love to work on having this PR merged.

Reply:
Anyone who has installed this package can have done `from semantic_release import UnknownCommitMessageStyleError` and next release after this will break this so thus we need a major release in order to do it.